src/ILCompiler.Compiler/src/Compiler/DependencyAnalysis/ObjectNode.Sorting.cs

```diff
@@ -0,0 +1,144 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Might be nice to decide whether we want dot files for sorting and stick to the choice. It feels inconsistent to have a dot file here, #if SUPPORT_JIT elsewhere, and no ability to make it optional elsewhere.

I would put it under Support Jit. We'll need to come up with a good scheme for JIT anyway and this will just be one more thing to clean up.